### DISCUSSION TITLE
Static Hosting on VSCode

### DISCUSSION THREAD

**Discussion by friskblack** [slidevjs/slidev#939] [General]
Is it possible to use LiveServer in VSCode for static hosting?

A simple build will give the following error

GET http://127.0.0.1:5501/assets/index.fbc75e81.js net::ERR_ABORTED 404 (Not Found)

127.0.0.1/:12 Refused to apply style from 'http://127.0.0.1:5501/assets/index.c055832a.css' because its MIME type ('text/html') is not a supported stylesheet MIME type, and strict MIME checking is enabled.

**Comment by mogador26** [comment]
Hello,
I think we have the same one with different execution contexts.
I'm using version 0.50.0-beta.5 of slidev with a deployment on github pages. When I display the site on github.io I get the following error on several .js files that I can't resolve:

github.io/assets/index-Bcb5a6Mv.js‘ has been blocked due to a forbidden MIME type (’text/html").

Do you have any ideas for solving this type of problem?

**Reply by kermanx** [reply]
You may need a [`--base` option](https://sli.dev/guide/hosting#base) to solve this.

There is also a GitHub workflow you can use to deploy your slides: https://sli.dev/guide/hosting#github-pages
